oat-sa/tao-test-runner-qti-fe
# Feature/tca 549/jump links additional funcs 1

**Related**
https://oat-sa.atlassian.net/browse/TCA-549

**Description**
Jumpl links changes. For more information please have a look at the [Jira ticket](https://oat-sa.atlassian.net/browse/TCA-549) _Acceptance Criteria_ section

**How to test**
1. Create test with several section. Some of them should be without review panel
2. Start test as test taker
3. Check Jump links
4. Switch between sections and check status JL
5. Switch between items, answer for some of them and check Question JL
6. Try to flag / unflag item and check question JL again
